### DISCUSSION TITLE
How do I add custom Intellisense?

### DISCUSSION THREAD

**Discussion by Allogeness** [James-Yu/LaTeX-Workshop#4232] [Q&A] [answered]
When I was writing my graduation thesis, I used a template made by my senior that met the school's requirements. There is a \citing{} command in it, which can add a literature citation in the upper right corner. I want to add this command to IntelliSense
![屏幕截图 2024-04-14 013402](https://github.com/James-Yu/LaTeX-Workshop/assets/108565991/1a85819e-a48e-4f63-bec1-6d37b07ce466)
, but it cannot automatically prompt the references in the reference.bib like \cite{}. 
![屏幕截图 2024-04-14 013321](https://github.com/James-Yu/LaTeX-Workshop/assets/108565991/f7418cec-ab39-477e-a554-58d507026d52)
![屏幕截图 2024-04-14 013348](https://github.com/James-Yu/LaTeX-Workshop/assets/108565991/7dfabdc5-559f-415a-bc06-d20db5e43706)
How can I modify it?

**Comment by James-Yu** [comment, marked as answer]
You can redefine the macro to another name starting with “cite”.
